packages/app-types/index.d.ts

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
Unrelated to this file, can you remove this hack from packages/firestore/test/integration/util/firebase_export.ts:

```
// TODO(b/66917182): This "as any" removes all of our type-checking in tests and
// is therefore pretty bad. But I can't figure out how to avoid it right now.
export default firebase as any;
```

With this refactor, the "as any" is no longer required.

(and then you can close http://b/66917182)

Reply:
Gotcha!